lib/healthcheck/government_data_check.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
```

Comment:
You don't need these anymore since https://github.com/alphagov/content-publisher/pull/1769

Reply:
Didn't realise that'd happened, nice! Thanks 👍